File: Day_21/Q84.cpp
#include <bits/stdc++.h>
using namespace std;
//Converting The String To Upper Case
void to_upper_case(string &str){
    for(int i = 0;i < str.length();i++){
        char ch = str[i];
        if((int)ch >= 97 && (int)ch <= 122){
            str[i] = ch - 32; //Converting To UpperCase By Using Ascii Values
        }
    }
}
int main(){
    string str;
    cin >> str;
    to_upper_case(str);
    //Printing The Result
    cout << str;
    return 0;
}

File: Day_21/Q84.java
import java.util.*;
class Q84{
    //Converting The String To Upper Case
    static void to_upper_case(StringBuilder str){
        for(int i = 0;i < str.length();i++){
            char ch = str.charAt(i);
            if(ch >= 97 && ch <= 122){
                char up = (char)(ch-32);
                str.setCharAt(i,up); //Converting To UpperCase By Using Ascii Values
            }
        }
    }
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        String str = sc.nextLine();
        StringBuilder sb = new StringBuilder(str);
        to_upper_case(sb);
        str = sb.toString();
        //Printing The Result
        System.out.print(str);
        sc.close();
    }
}